.github/workflows/sha1.yml

```diff
@@ -137,11 +137,10 @@ jobs:
 
   # Cross-compiled tests
   cross:
-    needs: set-msrv
     strategy:
       matrix:
         rust:
-          - ${{needs.set-msrv.outputs.msrv}}
+          - 1.51 # 1.41-1.50 `--features` can't be used inside virtual manifest
```

Comment:
This was the reason why to change the directory and not to use the package option [1].

[1] https://github.com/RustCrypto/hashes/pull/351/commits/1902274667add95b2fe469c2e184f42a1c1aa7cd#diff-7989ad9cb1e81934c64205efcfae23eb8bfc6aa827cc5c0f992a9704687a56e8L31

Reply:
Addition of this option is intentional. It is needed in cases when we prototype stuff using `[patch.crates-io]` in workspace's Cargo.toml.